Finance/paying-invoices.md

```diff
@@ -0,0 +1,19 @@
+# Paying Invoices
+
+Prior to paying the first invoice for a contractor, confirm with the contractor whether they would like to pay in CAD, USD, EUR, or GBP, and obtain their relevant banking details.
```

Comment:
```suggestion
Prior to paying the first invoice for a contractor, confirm with the contractor and project lead in what currency they would be paid in (CAD, USD, EUR, or GBP) then obtain their relevant banking details.
```

Don't think we want to make this a free for all. It gets complicated when we pay people with currency other whats on their invoice. I think this should be a decision in collaboration with the project lead since currency we get paid in also matters.